docs/guides/portals/sickle-cell.md

```diff
@@ -1,51 +1,10 @@
-# Index
-### [Glossary](#Glossary)
-### [Visualizations](#Visualizations)
-####  &nbsp;&nbsp;&nbsp;&nbsp; [Genome Browser](#genome-browser)
-#####  &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;[Tutorial](#genomeBrowserTutorial)
-####  &nbsp;&nbsp;&nbsp;&nbsp; [Variants and Phenotype Viewer](#variantsPhenotypeViewer)
-#####  &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;[Tutorial](#variantsPhenoyypeViewerTutorial)
+The Sickle Cell Genomics Portal contains two viewers for the exploration of data from the  [Sickle Cell Genome Project (SGP)](../data/data-access-units.md#sickle-cell-genome-project-(sgp)) dataset.
```

Comment:
Link should point to `../data/data-access-units.md#sickle-cell-genome-project-sgp`.